### PR TITLE
admin functions created

### PR DESCRIPTION
**Pull request by rahulambastha**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Hospital</title>
```
Comment (Prathviraj-B-N) [inline public/index.html:27]: This must be hospital

File: src/App.js [hunk 2]
```diff
@@ -1,23 +1,30 @@
...
 import './App.css';
-import { Header } from './components';
-import { Login, Receptionist, RegisterPatient, ViewPatients } from './features';
+import {Entry} from './entry'
+import { DoctorRegister } from './features/Doctor/doctorRegister/doctorregister';
+import { Receptionist } from './features';
+import { useState } from "react"
+import {DoctorList} from './Doctorview';
+
+import {Button } from './components'
```
Comment (Prathviraj-B-N) [inline src/App.js:8]: optimize imports like this import { Header,Button } from './components';

File: src/components/Header/Header.js [hunk 3]
```diff
@@ -1,20 +1,19 @@
 import React from 'react';
 import { Nav, Navbar } from 'react-bootstrap';
 import './Header.css'
-import { constants } from '../../constants';
 
 function Header() {
     return (
         <Navbar expand="lg" className='navbar-bg' >
-          <Navbar.Brand className='navbar-brand-style'>{constants.REACT_APP_NAV_BRAND}</Navbar.Brand>
+          <Navbar.Brand className='navbar-brand-style'>{process.env.REACT_APP_NAV_BRAND}</Navbar.Brand>
```
Comment (Prathviraj-B-N) [inline src/components/Header/Header.js:8]: constants in process.env has been moved to constants.js so change all process.env. to constants proces.env.

File: src/App.js [hunk 4]
```diff
@@ -1,23 +1,30 @@
...
+import {Route, Routes, useNavigate} from "react-router-dom";
 
 function App() {
+
   return (
```
Comment (Prathviraj-B-N) [inline src/App.js:13]: resolve routing and entry point

File: src/features/Doctor/doctorRegister/doctorregister.css [hunk 5]
```diff
@@ -0,0 +1 @@
+
```
Comment (Prathviraj-B-N) [inline src/features/Doctor/doctorRegister/doctorregister.css:1]: remove this file

**Review by Prathviraj-B-N — CHANGES_REQUESTED** [review]

File: src/App.css [hunk 6]
```diff
@@ -1,7 +1,22 @@
 .App {
   text-align: center;
 }
+.inputfiled{
+ margin-top: 100px;
+}
+.radio{    text-align: left;
+  }
+input[type="radio"]{
```
Comment (rohitraman) [inline src/App.css:9]: Alignment issues

File: src/App.css [hunk 7]
```diff
@@ -1,7 +1,22 @@
...
+  display: inline-block;
+    width: 10%;
+    height: 2em;
+    border-radius: 200%;
+  margin: 80px 0 100px  0;
+  accent-color: rgb(6, 2, 32);
 
+ 
+}
+.label{font-size: 40px;}
```
Comment (rohitraman) [inline src/App.css:19]: Alignment issues

File: src/App.js [hunk 8]
```diff
@@ -1,23 +1,30 @@
...
+   <Routes>
+
+<Route path='/' element={<Entry />} />
+<Route path='/docregister' element={<DoctorRegister />} />
+<Route path='/docregister/view' element={<DoctorList />} />
+
+
+
+   </Routes>
+
```
Comment (rohitraman) [inline src/App.js:23]: Take a pull of main

File: src/components/OptionsBox/OptionsBox.css [hunk 9]
```diff
@@ -1,8 +0,0 @@
-.landing-page-box {
```
Comment (rohitraman) [inline src/components/OptionsBox/OptionsBox.css:1]: Why is this file deleted

File: src/components/OptionsBox/OptionsBox.js [hunk 10]
```diff
@@ -1,16 +0,0 @@
-import React from 'react';
-import { useNavigate } from 'react-router-dom';
```
Comment (rohitraman) [inline src/components/OptionsBox/OptionsBox.js:2]: Why is this file deleted?

File: src/components/Sidebar/Sidebar.js [hunk 11]
```diff
@@ -1,29 +1,28 @@
...
+          <SidebarItem imgName = {process.env.REACT_APP_SIDEBAR_RECENT_CONSENTS_IMG} alt={process.env.REACT_APP_SIDEBAR_RECENT_CONSENTS_IMG} sidebarText={process.env.REACT_APP_SIDEBAR_RECENT_CONSENTS} />
           <div className='sidebar-divider' />
 
           <div className='sidebar-emergency-style'>
             <button className='sidebar-emergency-button'>
-                <img src={process.env.PUBLIC_URL + `/${constants.REACT_APP_SIDEBAR_WARNING_IMG}.png`} className='sidebar-emergency-image' alt={constants.REACT_APP_SIDEBAR_WARNING_IMG} />
-                <span className='sidebar-emergency-text'>{constants.REACT_APP_SIDEBAR_EMERGENCY_BUTTON_TEXT}</span>
+                <img src={process.env.PUBLIC_URL + `/${process.env.REACT_APP_SIDEBAR_WARNING_IMG}.png`} className='sidebar-emergency-image' alt={process.env.REACT_APP_SIDEBAR_WARNING_IMG} />
+                <span className='sidebar-emergency-text'>{process.env.REACT_APP_SIDEBAR_EMERGENCY_BUTTON_TEXT}</span>
             </button>
```
Comment (rohitraman) [inline src/components/Sidebar/Sidebar.js:26]: Moved from .env to constants, please take a pull from main

File: src/constants.js [hunk 12]
```diff
@@ -1,39 +0,0 @@
-export const constants  = {
-// Labels to be shown
-REACT_APP_NAV_BRAND : "Hospital",
-REACT_APP_SIDEBAR_SEARCH_PATIENT_DATA : "Search Patient Data",
-REACT_APP_SIDEBAR_REQUEST_CONSENT : "Request Consent",
```
Comment (rohitraman) [inline src/constants.js:5]: Take a pull of main
